cmd/vulcan-aws-alerts/main.go

```diff
@@ -70,8 +70,8 @@ func getCredentials(url string, accountID, role string) (*credentials.Credential
 	if role != "" {
 		m["role"] = role
 	}
-	jsonBody, err := json.Marshal(m)
-	req, err := http.NewRequest("POST", url, bytes.NewBuffer(jsonBody))
+	jsonBody, _ := json.Marshal(m)
+	req, _ := http.NewRequest("POST", url, bytes.NewBuffer(jsonBody))
```

Comment:
I guess you are removing the `err` because we were not checking it, but then I'd suggest checking the errors instead.

Reply:
Ok, I didn't do because the input parameters are well known and it's unlikely that happend. But agree it's easy and better to do it. Thanks!